### ISSUE TITLE
[Bug]: Gemini streaming response returning 'content': None on enabling of usage 

### ISSUE THREAD

**Issue by pradeepsinghjaroliya** [BerriAI/litellm#4823]
### What happened?

I'm trying to use Gemini models like this:
```
from litellm import completion
gemini_response = await acompletion(**final_payload)
return StreamingResponse(
                stream_chat_completion(gemini_response),
                media_type="text/event-stream",
            )
```
it's working great and streaming too, but in the response chunk, I don't see the usage or usageMetadata property
I tried adding 
`final_payload['stream_options'] = {"include_usage": True}`
in payload like openai , now the response has usage but the content of all streamed chunks is None
```
{'id': 'chat...., 'choices': [
        {'finish_reason': None, 'index': 0, 'delta': {'content': None, 'role': None, 'function_call': None, 'tool_calls': None
            }, 'logprobs': None
        }
    ], 'created': 1721667333, 'model': 'gemini-pro', 'object': 'chat.completion.chunk', 'system_fingerprint': None
}
```
this is the last chunk , it have usage
```
{'id': 'chat....', 'choices': [
        {'finish_reason': None, 'index': 0, 'delta': {'content': None, 'role': None, 'function_call': None, 'tool_calls': None
            }, 'logprobs': None
        }
    ], 'created': 1721667333, 'model': 'gemini-pro', 'object': 'chat.completion.chunk', 'system_fingerprint': None, 'usage': {'completion_tokens': 305, 'prompt_tokens': 32, 'total_tokens': 337
    }
}
```

### Twitter / LinkedIn details

https://www.linkedin.com/in/pradeep-singh-j-b2b66b14b/

**Comment by ghost** [comment]
Hi @pradeepsinghjaroliya can you confirm you see this issue on the latest version of litellm - v`1.41.26` ?

**Comment by ghost** [comment]
Unable to repro @pradeepsinghjaroliya 

Just ran a streaming call for gemini 

Call: 
```bash
curl -X POST 'http://0.0.0.0:4000/chat/completions' \
-H 'Content-Type: application/json' \
-H 'Authorization: Bearer sk-1234' \
-D '{
    "model": "gemini-1.5-flash",
    "messages": [
        { "role": "system", "content": "Use your tools smartly"},
        { "role": "user", "content": "What time is it now? Use your tool"}
    ],
    "stream": true,
    "stream_options": {"include_usage": true}
}'
```

```
data: {"id":"chatcmpl-fac51d72-e478-4394-9762-8e0f21e9f843","choices":[{"index":0,"delta":{"content":"I","role":"assistant"}}],"created":1721705078,"model":"gemini-1.5-flash","object":"chat.completion.chunk"}

data: {"id":"chatcmpl-fac51d72-e478-4394-9762-8e0f21e9f843","choices":[{"index":0,"delta":{"content":" do not have access to real-time information, including the current time. To"}}],"created":1721705078,"model":"gemini-1.5-flash","object":"chat.completion.chunk"}

data: {"id":"chatcmpl-fac51d72-e478-4394-9762-8e0f21e9f843","choices":[{"index":0,"delta":{"content":" get the current time, I recommend checking a clock, your phone, or an"}}],"created":1721705078,"model":"gemini-1.5-flash","object":"chat.completion.chunk"}

data: {"id":"chatcmpl-fac51d72-e478-4394-9762-8e0f21e9f843","choices":[{"index":0,"delta":{"content":" online time service. \n"}}],"created":1721705079,"model":"gemini-1.5-flash","object":"chat.completion.chunk"}

data: {"id":"chatcmpl-fac51d72-e478-4394-9762-8e0f21e9f843","choices":[{"finish_reason":"stop","index":0,"delta":{}}],"created":1721705079,"model":"gemini-1.5-flash","object":"chat.completion.chunk"}

data: {"id":"chatcmpl-fac51d72-e478-4394-9762-8e0f21e9f843","choices":[{"index":0,"delta":{}}],"created":1721705079,"model":"gemini-1.5-flash","object":"chat.completion.chunk","usage":{"completion_tokens":37,"prompt_tokens":14,"total_tokens":51}}

data: [DONE]
```

**Comment by ghost** [comment]
works without issues

**Comment by ghost** [comment]
closing as unable to repro. i've dm'ed on linkedin - let me know if the issue persists.

**Comment by pradeepsinghjaroliya** [comment]
Thanks, issue was fixed on updating to v1.41.26
